x/payment/genesis_test.go

```diff
@@ -0,0 +1,14 @@
+package payment_test
+
+// func TestGenesis(t *testing.T) {
+// 	genesisState := types.GenesisState{
+// 		// this line is used by starport scaffolding # genesis/test/state
+// 	}
+
+// 	k, ctx := keepertest.PaymentKeeper(t)
+// 	payment.InitGenesis(ctx, *k, genesisState)
+// 	got := payment.ExportGenesis(ctx, *k)
+// 	require.NotNil(t, got)
+
+// 	// this line is used by starport scaffolding # genesis/test/assert
+// }
```

Comment:
Is it caused by pulling from upstream or we just don't need this test in the future?

Reply:
this was not needed, but I forgot to delete, thanks! https://github.com/celestiaorg/lazyledger-app/pull/127/commits/0d0b88a989b182b3876cbb501291a9b6f798f2d1